banyand/measure/measure.go

```diff
@@ -70,10 +73,14 @@ func (s *measure) Close() error {
 	return s.indexWriter.Close()
 }
 
-func (s *measure) parseSpec() {
+func (s *measure) parseSpec() (err error) {
 	s.name, s.group = s.schema.GetMetadata().GetName(), s.schema.GetMetadata().GetGroup()
 	s.entityLocator = partition.NewEntityLocator(s.schema.GetTagFamilies(), s.schema.GetEntity())
 	s.maxObservedModRevision = pbv1.ParseMaxModRevision(s.indexRules)
+	if s.schema.Interval != "" {
```

Comment:
It is possible that the `Interval` is empty?

Reply:
absolutely, the interval of `service_traffic` is empty.

Reply:
> absolutely, the interval of `service_traffic` is empty.

No. I suppose for `service_traffic`, it is `Minute`.

Reply:
@lujiajing1126 All `*Traffic` data doesn't put time dimensionality into the ID. Basically, traffic is only TTL relative.

Reply:
> @lujiajing1126 All `*Traffic` data doesn't put time dimensionality into the ID. Basically, traffic is only TTL relative.

Is that because we don't have any `field` for `*Traffic`? 

Since interval is mostly for GORILLA encoding, it is meaningless in such scenarios

Reply:
The interval is bound to the field encoding algorithm, not to the TTL.